Investigate error in posting an event

```
com.exceptionless.exceptionlessclient.exceptions.SubmissionClientException: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class io.netty.buffer.ByteBufInputStream and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: java.util.ArrayList[0]->com.exceptionless.exceptionlessclient.submission.EventRequest["data"]->java.util.HashMap["@ext"]->org.springframework.core.codec.CodecException["cause"]->com.fasterxml.jackson.databind.exc.InvalidDefinitionException["location"]->com.fasterxml.jackson.core.JsonLocation["sourceRef"])
	at com.exceptionless.exceptionlessclient.submission.DefaultSubmissionClient.postSubmission(DefaultSubmissionClient.java:89) ~[exceptionless-client-1.0-beta1.jar:na]
	at com.exceptionless.exceptionlessclient.submission.DefaultSubmissionClient.postEvents(DefaultSubmissionClient.java:52) ~[exceptionless-client-1.0-beta1.jar:na]
	at com.exceptionless.exceptionlessclient.queue.DefaultEventQueue.process(DefaultEventQueue.java:140) ~[exceptionless-client-1.0-beta1.jar:na]
	at com.exceptionless.exceptionlessclient.queue.DefaultEventQueue.onProcessQueue(DefaultEventQueue.java:80) ~[exceptionless-client-1.0-beta1.jar:na]
	at com.exceptionless.exceptionlessclient.queue.DefaultEventQueue$1.run(DefaultEventQueue.java:65) ~[exceptionless-client-1.0-beta1.jar:na]
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556) ~[na:na]
	at java.base/java.util.TimerThread.run(Timer.java:506) ~[na:na]
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class io.netty.buffer.ByteBufInputStream and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: java.util.ArrayList[0]->com.exceptionless.exceptionlessclient.submission.EventRequest["data"]->java.util.HashMap["@ext"]->org.springframework.core.codec.CodecException["cause"]->com.fasterxml.jackson.databind.exc.InvalidDefinitionException["location"]->com.fasterxml.jackson.core.JsonLocation["sourceRef"])
	at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:77) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.SerializerProvider.reportBadDefinition(SerializerProvider.java:1277) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.DatabindContext.reportBadDefinition(DatabindContext.java:400) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.failForEmpty(UnknownSerializer.java:71) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.impl.UnknownSerializer.serialize(UnknownSerializer.java:33) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:755) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:755) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:755) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serializeFields(MapSerializer.java:726) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serializeWithoutTypeInfo(MapSerializer.java:681) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:637) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:33) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:755) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serializeContents(IndexedListSerializer.java:119) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:79) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:18) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4407) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3661) ~[jackson-databind-2.11.2.jar:2.11.2]
	at com.exceptionless.exceptionlessclient.submission.DefaultSubmissionClient.postSubmission(DefaultSubmissionClient.java:70) ~[exceptionless-client-1.0-beta1.jar:na]
	... 6 common frames omitted

```

--- a/src/main/java/com/exceptionless/exceptionlessclient/utils/Utils.java
+++ b/src/main/java/com/exceptionless/exceptionlessclient/utils/Utils.java
@@ -17,6 +17,7 @@ public final class Utils {
     JSON_MAPPER = new ObjectMapper();
     JSON_MAPPER.registerModule(new JavaTimeModule());
     JSON_MAPPER.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    JSON_MAPPER.disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
   }
 
   private static final Logger LOG = LoggerFactory.getLogger(Utils.class);

--- a/src/test/java/com/exceptionless/exceptionlessclient/ExceptionlessClientTest.java
+++ b/src/test/java/com/exceptionless/exceptionlessclient/ExceptionlessClientTest.java
@@ -1,18 +1,18 @@
 package com.exceptionless.exceptionlessclient;
 
 import com.exceptionless.exceptionlessclient.configuration.ConfigurationManager;
-import com.exceptionless.exceptionlessclient.models.Event;
-import com.exceptionless.exceptionlessclient.models.UserDescription;
 import com.exceptionless.exceptionlessclient.enums.EventPropertyKey;
 import com.exceptionless.exceptionlessclient.enums.EventType;
-import com.exceptionless.exceptionlessclient.settings.SettingsResponse;
-import com.exceptionless.exceptionlessclient.submission.SubmissionResponse;
+import com.exceptionless.exceptionlessclient.models.Event;
+import com.exceptionless.exceptionlessclient.models.UserDescription;
 import com.exceptionless.exceptionlessclient.queue.DefaultEventQueue;
 import com.exceptionless.exceptionlessclient.settings.DefaultSettingsClient;
 import com.exceptionless.exceptionlessclient.settings.ServerSettings;
+import com.exceptionless.exceptionlessclient.settings.SettingsResponse;
 import com.exceptionless.exceptionlessclient.storage.InMemoryStorage;
 import com.exceptionless.exceptionlessclient.storage.InMemoryStorageProvider;
 import com.exceptionless.exceptionlessclient.submission.DefaultSubmissionClient;
+import com.exceptionless.exceptionlessclient.submission.SubmissionResponse;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;

--- a/src/test/java/com/exceptionless/exceptionlessclient/utils/UtilsTest.java
+++ b/src/test/java/com/exceptionless/exceptionlessclient/utils/UtilsTest.java
@@ -1,5 +1,6 @@
 package com.exceptionless.exceptionlessclient.utils;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import org.junit.jupiter.api.Test;
 
 import java.net.URI;
@@ -79,5 +80,12 @@ public class UtilsTest {
   public void itCanMatchAValueToAPattern() {
     assertThat(Utils.match("abc", "def")).isFalse();
     assertThat(Utils.match("ABC", "abc")).isTrue();
+  }
+
+  @Test
+  public void itCanSerializeEmptyBeans() throws JsonProcessingException {
+    class EmptyBean {}
+
+    assertThat(Utils.JSON_MAPPER.writeValueAsString(new EmptyBean())).isEqualTo("{}");
   }
 }

